From e92e7c704694f76a8ae915fcbdd5079c87427af2 Mon Sep 17 00:00:00 2001
From: Stephen Cheng <stephen.cheng@cloud.com>
Date: Mon, 29 Jul 2024 02:33:13 +0100
Subject: [PATCH] CP-46112: [build] uninitialized variable issue

Signed-off-by: Stephen Cheng <stephen.cheng@cloud.com>
---
 src/tests/bigint_test.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/tests/bigint_test.c b/src/tests/bigint_test.c
index 75a80622..c4896a9b 100644
--- a/src/tests/bigint_test.c
+++ b/src/tests/bigint_test.c
@@ -206,7 +206,7 @@ void bigint_mod_exp_sample ( const bigint_element_t *base0,
 	static const uint8_t addend_raw[] = addend;			\
 	static const uint8_t value_raw[] = value;			\
 	static const uint8_t expected_raw[] = expected;			\
-	uint8_t result_raw[ sizeof ( expected_raw ) ];			\
+	uint8_t result_raw[ sizeof ( expected_raw ) ] = {0};			\
 	unsigned int size =						\
 		bigint_required_size ( sizeof ( value_raw ) );		\
 	bigint_t ( size ) addend_temp;					\
-- 
2.45.1

