From 76ec2a0540b25dbd183b9ce185583a4b24278cf1 Mon Sep 17 00:00:00 2001
From: Michael Brown <mcb30@ipxe.org>
Date: Wed, 4 May 2016 15:54:10 +0100
Subject: [PATCH] [skge] Fix building with GCC 6

Signed-off-by: Michael Brown <mcb30@ipxe.org>
---
 src/drivers/net/skge.c | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/src/drivers/net/skge.c b/src/drivers/net/skge.c
index 6384e764..c3264225 100755
--- a/src/drivers/net/skge.c
+++ b/src/drivers/net/skge.c
@@ -84,9 +84,6 @@ static struct net_device_operations skge_operations = {
 /* Avoid conditionals by using array */
 static const int txqaddr[] = { Q_XA1, Q_XA2 };
 static const int rxqaddr[] = { Q_R1, Q_R2 };
-static const u32 rxirqmask[] = { IS_R1_F, IS_R2_F };
-static const u32 txirqmask[] = { IS_XA1_F, IS_XA2_F };
-static const u32 napimask[] = { IS_R1_F|IS_XA1_F, IS_R2_F|IS_XA2_F };
 static const u32 portmask[] = { IS_PORT_1, IS_PORT_2 };
 
 /* Determine supported/advertised modes based on hardware.
@@ -1922,8 +1919,6 @@ static void skge_tx_clean(struct net_device *dev)
 	skge->tx_ring.to_clean = e;
 }
 
-static const u8 pause_mc_addr[ETH_ALEN] = { 0x1, 0x80, 0xc2, 0x0, 0x0, 0x1 };
-
 static inline u16 phy_length(const struct skge_hw *hw, u32 status)
 {
 	if (hw->chip_id == CHIP_ID_GENESIS)
-- 
2.45.1

